src/Frontend/vitest.config.ts

```diff
@@ -16,7 +16,7 @@ export default defineConfig({
     ],
   },
   test: {
-    pool: "forks", //https://github.com/vitest-dev/vitest/issues/2008#issuecomment-187106690
+    pool: "forks",
```

Comment:
`forks` is now the default.

Reply:
If forks is now the default, do we still need to explicitly configure it?